tests/schemas/test_user.py

```diff
@@ -0,0 +1,9 @@
+from pipeline.schemas.user import UserCreate
+
+
+def test_user_create_optional_username():
+    none_username_payload = dict(
+        email="email@company.com", password="ExamplePass123", username=None
+    )
+    UserCreate(**none_username_payload)
+    assert True
```

Comment:
Don't think you really need this line since it's not really doing anything.

Not sure if we already have some other tests for the validator but might be worth adding one test for a valid username and one test for an invalid one if it's quick to do?

Reply:
Thanks man! Removed the `assert True`and added some tests for valid + invalid usernames.